src/pages/dashboard/DashboardPage.tsx

```diff
@@ -84,7 +84,7 @@ const DashboardPage = () => {
             checked={groupByHour}
             onChange={(e) => setGroupByHour(e.target.checked)}
           />
-          Group by hour
+          {TEXTS.group_by_hour_tooltip_content}
```

Comment:
definitely unrelated, but does this checkbox effects anything but the lower widget? If not, I'll open an issue to move it down there

Reply:
I believe it should effect everything, but I think it's not working well